applications/nrf_desktop/src/modules/ble_conn_params.c

```diff
@@ -31,17 +35,30 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BLE_CONN_PARAMS_LOG_LEVEL);
 
 #define CONN_PARAMS_ERROR_TIMEOUT	K_MSEC(100)
 
+#ifdef CONFIG_DESKTOP_BLE_MANAGE_CI
+#define CONN_INTERVAL_BLE_SUSPENDED_REG ((CONFIG_DESKTOP_BLE_CI_SUSPENDED_MS * 1000U) / 1250U)   /* REG(MS) = MS / 1,25 ms */
```

Comment:
You could use MSEC_PER_SEC to be more explicit here.

Reply:
As we are speeking about us and us, USEC_PER_MSEC is better. Will use this.

Reply:
Definitely. thanks :)